cumulusci/tasks/push/tasks.py

```diff
@@ -1,12 +1,13 @@
-import time
 from datetime import datetime
+import time
 
 from cumulusci.core.tasks import BaseTask
-from cumulusci.tasks.salesforce import BaseSalesforceApiTask
 from cumulusci.tasks.push.push_api import SalesforcePushApi
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
 
 class BaseSalesforcePushTask(BaseSalesforceApiTask):
-    completed_statuses = ['Succeeded','Failed','Cancelled']
+    completed_statuses = ['Succeeded', 'Failed', 'Cancelled']
```

Comment:
Good catch, but that is outside the scope of this PR (this was just a formatting change). I will file a new issue for this.